test/forkMainnet/TokenCollector.t.sol

```diff
@@ -120,7 +120,10 @@ contract TestTokenCollector is Addresses {
         bytes32 r,
         bytes32 s
     ) private pure returns (bytes memory) {
-        return abi.encode(TokenCollector.Source.Token, abi.encode(permit.owner, permit.spender, permit.amount, permit.deadline, v, r, s));
+        return
+            bytes.concat(
```

Comment:
Is it concatenating `abi.encodePacked(TokenCollector.Source.TokenPermit)` and `abi.encode(permit.owner, ...)`?

Reply:
Yes, actually it can be replaced by a single `abi.encodePacked`.

Reply:
0f31586